tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/backfill_overall_health_test.py

```diff
@@ -0,0 +1,191 @@
+"""
+Integration test for BackfillOverallHealth.
+"""
+# Python Imports
+import os
+
+# Third party imports
+from dateutil import parser
+
+# Project imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.backfill_overall_health import BackfillOverallHealth
+from common import JINJA_ENV, OBSERVATION, PERSON
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+
+OBSERVATION_TMPL = JINJA_ENV.from_string("""
+INSERT INTO `{{project}}.{{dataset}}.observation`
+(observation_id, person_id, observation_concept_id, observation_date, observation_type_concept_id, observation_source_concept_id)
```

Comment:
Could you add the `questionnaire_response_id` field? Acceptance criteria mentions there should be a difference for the observations that were backfilled vs a participant response.

Reply:
Sure! It always helps to make our tests more detailed so we can ensure our codes' behavior.